sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -1380,29 +1373,13 @@ public EditEventSteps(
         "^I click on New Sample and discard changes is asked$",
         () -> {
           webDriverHelpers.clickOnWebElementBySelector(NEW_SAMPLE_BUTTON);
-          if (webDriverHelpers.isElementVisibleWithTimeout(POPUP_DISCARD_CHANGES_BUTTON, 30)) {
-            webDriverHelpers.clickOnWebElementBySelector(POPUP_DISCARD_CHANGES_BUTTON);
-          }
-        });
-
-    When(
-        "^I confirm all actions until unsaved changes popup window disappears$",
-        () -> {
-          //          do {
-          //            WebElement element =
-          //                webDriverHelpers.returnTheVisibleAndClickableElement(CONFIRM_ACTION);
-          //            webDriverHelpers.clickOnWebElement(element);
-          //          } while (webDriverHelpers.isElementVisibleWithTimeout(CONFIRM_ACTION, 30));
         });
 
     When(
-        "^I confirm last popup window when there are multiple ones$",
```

Comment:
we can't update names of methods because they are used by other tests as well and this will cause failures

Reply:
its only used in this testcase